api-reference/50 Common/Object Structures/positionConfig/positionConfig.md

```diff
@@ -4,11 +4,99 @@ type: Object
 ---
 ---
 ##### shortDescription
-The **position** object specifies the widget positioning options.
+Configures the position of an overlay element.
 
 ---
-This object is passed to the **position** configuration option of a widget that overlays the main screen ([LoadPanel](/api-reference/10%20UI%20Widgets/dxLoadPanel '/Documentation/ApiReference/UI_Widgets/dxLoadPanel/'), [Popup](/api-reference/10%20UI%20Widgets/dxPopup '/Documentation/ApiReference/UI_Widgets/dxPopup/'), [Popover](/api-reference/10%20UI%20Widgets/dxPopover '/Documentation/ApiReference/UI_Widgets/dxPopover/'), and [Toast](/api-reference/10%20UI%20Widgets/dxToast '/Documentation/ApiReference/UI_Widgets/dxToast/')).
+This object should be assigned to the **position** option of an overlay widget ([Popup](/Documentation/ApiReference/UI_Widgets/dxPopup/), [Popover](/Documentation/ApiReference/UI_Widgets/dxPopover/), [Tooltip](/Documentation/ApiReference/UI_Widgets/dxTooltip/), etc.).
```

Comment:
Can we instead say:
Assign this object to the **position** option of an overlay widget